theme/templates/resource-landing-page/title-section.html

```diff
@@ -173,52 +190,138 @@ <h2 id="resource-title">{{ title }}</h2>
                         </li>
                     {% endif %}
                 {% endfor %}
-
             </ul>
         {% endif %}
     {% else %}
+        </div>
+        {# ======= Communities ======= #}
+        {% if communities_enabled %}
+            <div class="modal fade" id="title-modal" tabindex="-1" role="dialog" aria-labelledby="Title Assistant">
+                <div class="modal-dialog" role="document">
+                    <div id="title-assistant" class="modal-content">
+                        <div class="modal-body">
+                            <div>
+                                <textarea id="resource-title" v-model="title" disabled rows="3"
+                                          style="width: 100%; margin-bottom: 1em; cursor: not-allowed;"
```

Comment:
This inline style could be moved to some CSS rules in one of our files.

Reply:
Sure. Which one?

Reply:
This one:

> style="width: 100%; margin-bottom: 1em; cursor: not-allowed;"

Reply:
Which `one of our files` should the inline style go into

Reply:
We've been writing resource landing page CSS in here:
https://github.com/hydroshare/hydroshare/blob/develop/theme/static/css/hydroshare_core.css

Reply:
I can do that for now, but this goes against the Django collectstatic convention. Overriding this convention leads to colossal, hard to maintain files, similar to what we have in __init__.py in a few Django apps. I'll review with you when we get time on a voice call.

Reply:
In that case, feel free to use a CSS file within this app. We try not to put too much CSS inline in the HTML, just as long as it exists in some dedicated CSS file should be fine.

Reply:
Fixed in 81118c7c7